hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/impl/ZKFederationStateStoreOpDurations.java

```diff
@@ -54,25 +54,25 @@ public final class ZKFederationStateStoreOpDurations implements MetricsSource {
   private MutableRate registerSubClusterCall;
 
   @Metric("Duration for a deregister subCluster call")
-  private MutableRate deregisterSubCluster;
+  private MutableRate deregisterSubClusterCall;
```

Comment:
Do we need to add the Call at the end?

Reply:
Thanks for your suggestion, there is no need to add the word Call after the variable, I will modify the code.